test/PollFactory.js

```diff
@@ -0,0 +1,11 @@
+const { expect } = require("chai");
```

Comment:
More tests need to be added to this, like creating a poll, voting for an option, etc.